src/token_request_handler.ts

```diff
@@ -46,6 +51,22 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
     return (response as TokenErrorJson).error === undefined;
   }
 
+  performRevokeTokenRequest(
+      configuration: AuthorizationServiceConfiguration,
+      request: RevokeTokenRequest): Promise<boolean> {
+    let revokeTokenResponse = this.requestor.xhr<boolean>({
+      url: configuration.revocationEndpoint,
+      method: 'POST',
+      dataType: 'json',  // adding implicit dataType
+      headers: {'Content-Type': 'application/x-www-form-urlencoded'},
```

Comment:
Nitpick: Can you check if the `Content-Type` is set on the request as expected ? I am confused because the data is not being sent as a bunch of URL encoded parameters. It is being sent as JSON and `dataType` `json` will actually set the `Content-Type` to the `json` mime type. In which case you can remove this line in a subsequent PR.